ansible/roles_ocp_workloads/ocp4_workload_cloud_native_robot/defaults/main.yaml

```diff
@@ -0,0 +1,16 @@
+---
+become_override: false
+ocp_username: opentlc-mgr
+silent: false
+
+ocp4_workload_cloud_native_robot_user_count: >-
+  {{ ocp4_workload_authentication_htpasswd_user_count
+   | default(user_count)
+   | default(num_users)
+   | default(4)
+  }}
+ocp4_workload_cloud_native_robot_user_prefix: >-
+  {{ (ocp4_workload_authentication_htpasswd_user_base | default('team')) }}
```

Comment:
Why default to `team` instead of `user` like ocp4_workload_authentication role?

Reply:
Because the workshop works with teams and not users.